sandbox/technical_user_guide.md

```diff
@@ -136,17 +131,57 @@ Basic Mzg0MWM1OTQtYThjMC00MWU1LTk4Y2MtMzhiYjQ1MzYwZDNjOmY5NzgwZDMyMzU4OGYxY2RmYz
 * `Accept: application/json`
 * `Authorization: <Encoded Basic authentication>`
 
-**cURL command**
+**cURL commands**
+
+You can choose one of two cURL commands to use.
+
+**Option 1, which performs Basic authentication by passing in an additional header**
+
+Format:
```

Comment:
Is the format section necessary, since we have the example below?

Reply:
Keeping open, because this relates to the conversation in https://github.com/CMSgov/bcda-static-site/pull/57#discussion_r415877240

Reply:
Not strictly necessary, ie a person will intuit what to do by looking at the example, but I would like to maintain consistency with the language in *Option 2*, which gives a format then an example.

Reply:
Ignore ^^; accepted, resolved.